src/BsReactstrap__Form.re

```diff
@@ -1,32 +1,40 @@
-[@bs.module "reactstrap"] external form : ReasonReact.reactClass = "Form";
+[@bs.module "reactstrap"] external form: ReasonReact.reactClass = "Form";
 
 [@bs.obj]
-external makeProps : (
-  ~inline: bool=?,
-  ~tag: 'a=?,
-  ~innerRef: 'b=?,
-  ~className: string=?,
-  ~cssModule: 'c=?,
-  unit
-) => _ = "";
+external makeProps:
+  (
+    ~inline: bool=?,
+    ~onSubmit: 'a => unit=?,
```

Comment:
what is the shape of argument? is it a ReactEvent?

Reply:
Yes it is, I've made the change